src/locale.en.js

```diff
@@ -52,10 +57,11 @@ export const campsite = {
             none: 'None'
         },
         accessibility_options: {
+            blank: '',
             paved_road: "Paved Road",
             dirt_road: "Dirt Road",
             uneven_terrain: "Uneven Terrain",
-            fourbyfour: "4x4 Recommended",
+            fourbyfour: "High Clearance",
```

Comment:
Changed this to `High Clearance` to be a little clearer on what we meant by 'Recommended'. Let me know if you disagree @amgoodman5